Assignments/stu5/test-data-access.js

```diff
@@ -15,7 +15,7 @@ const main = async () => {
     // }
     // let personId = 1
 
-    let r = await dataAccess.getPerson();
+    let r = await dataAccess.getStockPersons();
```

Comment:
Should change the name for this to `getPersonsForType()`

Reply:
Did this get updated?

Reply:
I just did